com.unity.ml-agents/Runtime/DecisionRequester.cs

```diff
@@ -30,6 +39,18 @@ public class DecisionRequester : MonoBehaviour
         [FormerlySerializedAs("RepeatAction")]
         public bool TakeActionsBetweenDecisions = true;
 
+        /// <summary>
+        /// Whether or not the Agent decisions should start at an offset (different for each agent).
```

Comment:
I think this feature was removed as it was too complicated.

Reply:
Do you mean offsetStep? And if it were removed, shouldn't this field also be removed or deprecated?

Reply:
Is is removed on master

Reply:
Oh, I see. I must have accidentally added it back in when I merged master into this branch before making the PR.

Reply:
Hmm, maybe there are other things that were added back? Is there a way to be sure?

Reply:
Looking for code changes in this diff is probably the only way to be sure. 

Oddly, though, when I merged the latest master, that [change request diff](https://github.com/Unity-Technologies/ml-agents/pull/3803/commits/8b020038a15625cfb841ff88142fb79dc1524192#diff-fcf7fb2dae09e8a96b8746b0283afd87R52) shows offsetStep in both the master and the doc-review branch, so I really don't know what happened.